fbw-a380x/src/systems/systems-host/systems/PseudoFWC.ts

```diff
@@ -840,12 +841,19 @@ export class PseudoFWC {
   private readonly trueNorthRef = Subject.create(false);
 
   /* SURVEILLANCE */
-  private readonly gpwsFlaps3 = Subject.create(false);
 
-  private readonly gpwsFlapMode = Subject.create(0);
+  private readonly gpwsFlapModeOff = Subject.create(false);
+
+  private readonly gpwsSysOff = Subject.create(false);
+
+  private readonly gpwsGsOff = Subject.create(false);
 
   private readonly gpwsTerrOff = Subject.create(false);
 
+  private readonly xpdrAltReporting = ConsumerSubject.create(this.sub.on('mfd_xpdr_set_alt_reporting'), true); // fixme signal should come from XPDR?
```

Comment:
I see. I did some digging and on (some?) A320s it seems turning off the altitude reporting forces the transponder to operate in mode A but I have no idea if A380 is the same. I will change the logic to the same one as the MFD but we will have to change it at some point once the switching logic between XPDR SYS1 and SYS2 is in place as I assume it takes the active one for the memo (same for XPDR STBY)

Reply:
The next question is also how the sim (and VATSIM clients) handle alt rptg, I don't have much insight there :/
So yeah, let's keep the implementation the same for now (even if we're not sure whether it's the best way to go, and as soon as we have more references on the A380 let's change it. 
MSFS also seems to only support one transponder at the moment, so the SYS1/SYS2 implementation will be tricky

Reply:
I would implement a transponder manager that takes the states of the XPDR 1 & 2 LRUs and sets the MSFS state appropriately. As for what MSFS modes to use, the A32NX is correct here, and I thought we match that here unless it changed.

Reply:
> I would implement a transponder manager that takes the states of the XPDR 1 & 2 LRUs and sets the MSFS state appropriately from the combined state (similar to the AMUs). Best to keep real aircraft system simulation and host simulator stuff separated. As for what MSFS modes to use, the A32NX is correct here, and I thought we match that on the A380X unless it had been changed since we worked on it?

From my POV this change is outside the scope of this PR though, something which can be done in addition to the RMP PR after the MFD PR is merged. 
So I would keep the implementation as above for now (which is in line with how the MFD reads/writes the status).

Reply:
Yeah, I totally agree. XPDR work is for an XPDR systems PR.